app/components/Code/Code.less

```diff
@@ -3,14 +3,23 @@
 .root {
   .rawText(@standard-type-scale);
   position: relative;
-  padding: (@grid-row-height * 2) (@grid-gutter-width * 2);
+  padding: (@grid-row-height * 2) 0;
```

Comment:
This will render hard up against the blue border on the left. Maybe more suited to:
```
padding: (@grid-row-height * 2) 0 (@grid-row-height * 2) (@grid-gutter-width / 2)
```
![screen shot 2016-10-21 at 9 06 43 am](https://cloud.githubusercontent.com/assets/912060/19579699/b7f5eb7e-976d-11e6-89b7-5b668707d310.png)
Thoughts?